ivy_tests/test_ivy/test_frontends/test_torch/test_dropout_functions.py

```diff
@@ -50,3 +50,55 @@ def test_torch_dropout(
     for u in ret:
         # cardinality test
         assert u.shape == x.shape
+
+
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.dropout2d",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=50,
+        allow_inf=False,
+        min_num_dims=2,
+        max_num_dims=4,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    prob=helpers.floats(min_value=0, max_value=0.9),
+    training=st.booleans(),
+    data_format=st.sampled_from(["NCHW", "NHWC"]),
+    test_gradients=st.just(False),
+    test_with_out=st.just(False),
+)
+def test_torch_dropout2d(
+    *,
+    dtype_and_x,
+    prob,
+    training,
+    data_format,
+    test_flags,
+    backend_fw,
+    on_device,
+    fn_name,
+    ground_truth_backend,
+):
+    dtype, x = dtype_and_x
+    ret, gt_ret = helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=dtype,
+        test_flags=test_flags,
+        test_values=False,
+        on_device=on_device,
+        fw=backend_fw,
+        fn_name=fn_name,
+        x=x[0],
+        prob=prob,
+        training=training,
+        data_format=data_format,
+        return_flat_np_arrays=True,
```

Comment:
Don't think you need this here since this is handled in the `helpers.flatten_and_to_np` function right below. Can you elaborate on why you added this?

Reply:
you're right, I'll remove it.

> Can you elaborate on why you added this?

I didn't add, I took the [test for `ivy.dropout2d`](https://github.com/unifyai/ivy/commit/b51f9bc19f0ce2a3d3325109eddc8391333f772e#diff-951675de6359ce40933e5e8acc7a45effc4602b400264a202fa79fa7422915dfR574) and tried to adapt it as much as I can to my case, I thought that would be a more correct way of doing things, but it looks like I've missed a couple of things.

please let me know if the approach I took was problematic.